### PR TITLE
Ppu display

### PR DESCRIPTION
**Pull request by remicalixte**

### CODE REVIEW

File: ppu/display.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package ppu
+
+type display struct {
+	brightness  uint8 // Display brightness
+	forceBlank  bool  // If true, force screen to blank
+	vScanning   bool  // If true, interlace mode
+	objVDisplay bool  // \
+	bgVDisplay  bool  //  \
+	hPseudoMode bool  //    Some dank parameters
```
Comment (sfluor) [inline ppu/display.go]: Thoses comments

File: ppu/display.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	for i := 0; i < 4; i++ {
+		ppu.backgroundData.bg[i].subScreen = data&0x01 != 0
+		data = data >> 1
+	}
+	ppu.oam.subScreen = data&0x01 != 0
+	return 0
+}
+
+// 2133h - SETINI - Display Control 2 (W)
+func (ppu *PPU) setini(data uint8) uint8 {
```
Comment (sfluor) [inline ppu/display.go]: I think it would be better to directly use 0x02, 0x04, 0x08, etc. Instead of shifting data

File: ppu/display.go (hunk 1 above)
Comment (remicalixte) [reply ppu/display.go]: This is temporary

File: ppu/display.go (hunk 2 above)
Comment (remicalixte) [reply ppu/display.go]: Indeed but I think we should keep the loop

Comment (sfluor) [reply ppu/display.go]: which loop ?

Comment (remicalixte) [reply ppu/display.go]: the for loop

Comment (sfluor) [reply ppu/display.go]: Ah ! yes i was only speaking of setini register

Comment (remicalixte) [reply ppu/display.go]: done

File: ppu/display.go (hunk 1 above)
Comment (remicalixte) [reply ppu/display.go]: done !

**Pull request merged**